scripts/make_conf.py

```
@@ -0,0 +1,46 @@
+import shutil
```

Ruff (0.12.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)